packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -118,30 +118,41 @@ export class AccountForm extends PureComponent {
     if (locales && lang) {
       extend(locales[lang])
     }
+
+    if (props.oauth) return
+    this.sanitizedFields = Manifest.sanitizeFields(props.fields)
+    this.defaultValues = Manifest.defaultFieldsValues(this.sanitizedFields)
+  }
+
+  validate(vals) {
+    let errors = {}
+    for (let name in this.sanitizedFields)
+      if (this.sanitizedFields[name].required && !vals[name])
+        errors[name] = 'Required'
```

Comment:
We never use plain english in our code. We should here:
* Use a constant to identify the error.
or
* At least use a translated locale.